### DISCUSSION TITLE
[ObservableProperty] is not working in .net 4.8 framework

### DISCUSSION THREAD

**Discussion by kmostofi** [CommunityToolkit/dotnet#468] [Q&A]
Hi,
The project I am working on is targetting .netframework 4.8. 

The property field with ObervableProperty set as Private, however, the public property is not accessible. What am I missing here? 

It works if I use the nuget in .netcore environment. 

Please advice.

**Comment by rudderdon** [comment]
https://github.com/CommunityToolkit/dotnet/issues/695#issuecomment-1546899531

**Comment by Sergio0694** [comment]
If you're using `packages.config` and you're on the MVVM Toolkit 8.2.1, are you seeing a warning in our output?

**Comment by rudderdon** [comment]
Ok, I was able to get it to work, but I had to manually load the provided analyzers as well as set the language in my 4.8 csproj file to latest.

.csproj file update by adding:
<LangVersion>Latest</LangVersion>

I then right-clicked "Analyzers" under by project's "References" section and chose "Load Analyzer". The files I loaded were the "..\CommunityToolkit.Mvvm.8.2.0\analyzers\dotnet\roslyn4.0\cs" analyzers.

I then full rebuilt by project and the autogenerated classes started to show up and my automated bindable properties were accessible through my ObservableObject class.

![image](https://github.com/CommunityToolkit/dotnet/assets/1126103/d769ddc3-da12-45d9-947e-d3837f54c2df)
